app/controllers/api/vms_controller.rb

```diff
@@ -216,6 +216,16 @@ def disassociate_resource(type, id, data)
       end
     end
 
+    def resize_resource(type, id, data)
+      api_resource(type, id, "Resizing Resource", :supports => :resize) do |vm|
+        raise BadRequestError, "Must specify new resize value/s" if data["resizeValues"].blank?
```

Comment:
Not for this PR but it looks like none of these errors are translated, which we probably should do since it's a user facing message.

Reply:
@Fryguy is this a general API issue?

```
$ grep -r BadRequestError app/controllers/ | wc -l
252
$ grep -r BadRequestError app/controllers/ | grep '_(' | wc -l
1
```